scripts/run_nightly_tests_in_devicefarm_pool.sh

```diff
@@ -66,11 +66,79 @@ function stopDuplicates {
 }
 stopDuplicates
 
+# Select range of devices from version 7 and above.
+device1=$(aws devicefarm list-devices \
+                --region="us-west-2" \
+                --filters '[
+                    {"attribute":"AVAILABILITY","operator":"EQUALS","values":["HIGHLY_AVAILABLE"]},
+                    {"attribute":"PLATFORM","operator":"EQUALS","values":["ANDROID"]},
+                    {"attribute":"OS_VERSION","operator":"GREATER_THAN_OR_EQUALS","values":["7"]},
+                    {"attribute":"OS_VERSION","operator":"LESS_THAN","values":["7.1"]},
+                    {"attribute":"MANUFACTURER","operator":"IN","values":["Google", "Pixel", "Samsung"]}
+                ]' \
+                | jq -r '.devices[0].arn')
+
+device2=$(aws devicefarm list-devices \
+                --region="us-west-2" \
+                --filters '[
+                    {"attribute":"AVAILABILITY","operator":"EQUALS","values":["HIGHLY_AVAILABLE"]},
+                    {"attribute":"PLATFORM","operator":"EQUALS","values":["ANDROID"]},
+                    {"attribute":"OS_VERSION","operator":"GREATER_THAN_OR_EQUALS","values":["8"]},
+                    {"attribute":"OS_VERSION","operator":"LESS_THAN","values":["9"]},
+                    {"attribute":"MANUFACTURER","operator":"IN","values":["Samsung"]}
+                ]' \
+                | jq -r '.devices[0].arn')
+
+device3=$(aws devicefarm list-devices \
+                --region="us-west-2" \
+                --filters '[
+                    {"attribute":"ARN","operator":"NOT_IN","values":["'$device2'"]},
```

Comment:
looks like these might not be necessary given that `OS_VERSION` steps through numbers with no overlap?